packages/celotool/src/e2e-tests/governance_tests.ts

```diff
@@ -442,7 +442,7 @@ describe('governance tests', () => {
 
       // groupKit uses a different node than kit does, so wait a second in case kit's node
       // got the new block before groupKit's node did.
-      await sleep(1)
+      await sleep(5)
```

Comment:
Why was this increase necessary? Out of scope for this PR, but do we have any way to do an exact wait, rather than an arbitrary sleep?

Reply:
increased during debugging will revert